File: backend/src/main/java/org/globe42/dao/CoupleDao.java
package org.globe42.dao;

import org.globe42.domain.Couple;
import org.springframework.data.jpa.repository.JpaRepository;

/**
 * DAO for {@link org.globe42.domain.Couple}
 * @author JB Nizet
 */
public interface CoupleDao extends JpaRepository<Couple, Long> {
}

File: backend/src/main/kotlin/org/globe42/dao/CoupleDao.kt
package org.globe42.dao

import org.globe42.domain.Couple
import org.springframework.data.jpa.repository.JpaRepository

/**
 * DAO for [org.globe42.domain.Couple]
 * @author JB Nizet
 */
interface CoupleDao : JpaRepository<Couple, Long>
